__tests__/util/humanize-bytes.js

```diff
@@ -0,0 +1,49 @@
+/* @flow */
+import humanizeBytes from '../../src/util/humanize-bytes'
+
+const getLastChars = (str: string, num: number): string => {
+
+  let chars = [];
+
+  for (let i = 0; i < num; i++) {
+    chars.unshift(str[str.length - (i + 1)]);
+    
+  }
+
+  return chars.join('');
+}
+
+describe('humanizeBytes', () => {
+  test('handles 0, null, and undefined', () => {
+    expect(humanizeBytes(0)).toEqual('0B');
+    expect(humanizeBytes(null)).toEqual('0B');
+    expect(humanizeBytes(undefined)).toEqual('0B');
+  });
+
+  test('converts a buffer length to binary and outputs correct string', () => {
+    const bufferSize = 2000;
+    const buffer = Buffer.alloc(bufferSize);
+    const bufferLen = buffer.length;
+    expect(humanizeBytes(bufferLen)).toEqual('1.95KB');
+  });
+
+  test('appends the correct unit', () => {
+
+    const zero = 0;
+    const tenThousand = 10000;
+    const oneMillion = 1000000;
+    const oneHundredMillion = 100000000;
+    
+    const zeroHumanized = humanizeBytes(zero);
+    const tenThousandHumanized = humanizeBytes(tenThousand);
+    const oneMillionHumanized = humanizeBytes(oneMillion);
+    const oneHundredMillionHumanized = humanizeBytes(oneHundredMillion);
+    
+
+    expect(getLastChars(zeroHumanized, 1)).toEqual('B');
```

Comment:
Why not check the whole string to make sure the formatting and the numbers are also correct?